packages/@coorpacademy-components/src/variables/theme.native.ts

```diff
@@ -0,0 +1,120 @@
+type SpaceType = 'micro' | 'tiny' | 'small' | 'base' | 'medium' | 'large' | 'xlarge';
```

Comment:
Why theme has to move from mobile to component?
Isn't be possible to have different themes on different apps?

Reply:
this is a default typed theme; 
it can be injected from apps to use dedicated themes